client/src/containers/header/app/component.tsx

```diff
@@ -1,16 +1,34 @@
+import { useCallback } from 'react';
+
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 
 import cn from 'lib/classnames';
 
+import { menuOpenAtom } from 'store/explore-map';
```

Comment:
This shouldn't come from the explore-map store. If you want to have a global atom stored you may want to create a global store for all the pages